mavis/test/models.py

```diff
@@ -188,6 +197,21 @@ def year_groups(self) -> list[int]:
             case self.MENACWY | self.TD_IPV:
                 return list(range(9, 12))
 
+    @property
+    def tally_categories(self) -> list[str]:
+        common_categories = [
+            TallyCategory.NO_RESPONSE,
+            TallyCategory.CONTRAINDICATED_OR_DID_NOT_CONSENT,
```

Comment:
Worth noting that this will be changing shortly to only contain patients who did not consent.